src/site/static/eligibility.liquid

```diff
@@ -1871,8 +1871,29 @@ pageClass: "page-public-assistance"
   }
 
   function gaEligible() {
-    // TODO
-    return false;
+    // https://socialservices.sccgov.org/financial-assistance/financial-assistance-adults-without-dependent-children-general-assistance
```

Comment:
I recently found a nice source of eligibility documents for the Santa Clara County Social Services Agency.  I've been referencing those in the code whenever I can since those seem to be a better "source of truth" than some of the websites I originally linked to in the Benefits Eligibility spreadsheet.  It also allows for referencing specific section numbers of the policy.

Here are the GA-specific ones:
[GA policy handbook](https://socialservices.sccgov.org/about-us/department-employment-and-benefit-services/regulation-and-policy-handbooks/general-assistance)
[List of charts, including GA](https://socialservices.sccgov.org/about-us/department-employment-and-benefit-services/regulation-and-policy-handbooks/program-eligibility)